database/insertData.js

```diff
@@ -1,12 +1,35 @@
 const fs = require('fs')
+const path = require('path')
 const fetch = require('node-fetch')
 const config = require('../src/config.json')
 
+const dataFolder = './database/insertData'
+const sharedDataFiles = ['core_mutations.js', 'dev_actions.js']
+
 if (process.argv[2] === '--from_insert_data.sh') {
   const filesToProcess = fs
-    .readdirSync('./database/insertData')
+    .readdirSync(dataFolder, { withFileTypes: true })
+    .filter((dirent) => dirent.isFile())
+    .map((dirent) => dirent.name)
     .filter((file) => !file.match(/^\./)) // Ignore hidden files
-    .filter((file) => !['core_mutations.js', 'dev_actions.js'].includes(file))
+    .filter((file) => !sharedDataFiles.includes(file))
+
+  // Add locale-specific files
+  const localeFolder = process.argv[3]
+  if (localeFolder) {
+    console.log(`Locale: ${localeFolder}`)
+    const subfolderFilesToProcess = fs
+      .readdirSync(path.join(dataFolder, localeFolder))
+      .filter((file) => !file.match(/^\./)) // Ignore hidden files
+      .filter((file) => !sharedDataFiles.includes(file))
+    filesToProcess.push(
+      ...subfolderFilesToProcess.map((filename) => path.join(localeFolder, filename))
+    )
+    // Copy current version of core mutations/actions to subfolder
+    sharedDataFiles.forEach((file) =>
```

Comment:
why do we need to do this (can we not just import them from parent folder ? )

Reply:
I just wanted it to be possible easily move the insert mutations between the base level and subfolders freely, without having to remember to change the import path every time. I thought we might be doing it quite a bit...

Reply:
This is taken care of [here](https://github.com/openmsupply/application-manager-server/pull/390#issuecomment-853469430)